src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -126,20 +138,25 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
 
     // Ordering of the spread props is important: later styles override former ones
     return (
-      <a
-        id={platformProps?.id}
-        style={{
-          display: 'flex',
-          cursor: platformProps?.disabled ? 'default' : 'pointer',
-        }}
-        {...mouseHoverEvents}
-        onClick={platformProps?.disabled ? undefined : platformProps?.onClick}
-        onMouseDown={
-          platformProps?.disabled ? undefined : platformProps?.onMouseDown
-        }
+      <Tooltip
+        title={platformProps?.disabled ? undefined : title}
```

Comment:
Isn't it possible to wrap with the tooltip component only if the platform is not disabled?

Reply:
True though not sure which is cleaner